cclib/method/nuclear.py

```diff
@@ -9,12 +9,39 @@
 
 import logging
 
-import numpy
+import numpy as np
 
 from cclib.method.calculationmethod import Method
 from cclib.parser.utils import PeriodicTable
 
 
+def get_most_abundant_isotope(element):
+    """Given a `periodictable` element, return the most abundant
+    isotope.
+    """
+    most_abundant_isotope = element.isotopes[0]
+    abundance = 0
+    for iso in element:
+        if iso.abundance > abundance:
+            most_abundant_isotope = iso
+            abundance = iso.abundance
+    return most_abundant_isotope
+
+
+def get_isotopic_masses(charges):
+    """Return the masses for the given nuclei, respresented by their
+    nuclear charges.
+    """
+    import periodictable
```

Comment:
Shouldn't we import (softly) at top of module?

Reply:
Yes, I'll make it consistent with the bridges for now.